patches/server/0726-Expose-firework-item-directly-manually-setting-fligh.patch

```diff
@@ -0,0 +1,107 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Owen1212055 <23108066+Owen1212055@users.noreply.github.com>
+Date: Tue, 22 Jun 2021 23:41:11 -0400
+Subject: [PATCH] Expose firework item directly + manually setting flight ticks
+
+
+diff --git a/src/main/java/org/bukkit/craftbukkit/entity/CraftFirework.java b/src/main/java/org/bukkit/craftbukkit/entity/CraftFirework.java
+index be86114eac3975b82ca74d4d6ed3f0402a642e8a..3934a15dec286aa7d2130c8e8a53380a8e40590f 100644
+--- a/src/main/java/org/bukkit/craftbukkit/entity/CraftFirework.java
++++ b/src/main/java/org/bukkit/craftbukkit/entity/CraftFirework.java
+@@ -14,11 +14,12 @@ import org.bukkit.inventory.meta.FireworkMeta;
+ public class CraftFirework extends CraftProjectile implements Firework {
+ 
+     private final Random random = new Random();
+-    private final CraftItemStack item;
++    //private final CraftItemStack item; Paper - Expose firework item directly
+ 
+     public CraftFirework(CraftServer server, FireworkRocketEntity entity) {
+         super(server, entity);
+ 
++        /* Paper Start - Expose firework item directly
```

Comment:
I think we are discouraging the use of block comments nowadays. If upstream does a change inside the block comment, since the line doesnt conflict, it's not gonna be noticed. So change this to a bunch of inline comments `//`s